charts/capsule-proxy/templates/deployment.yaml

```diff
@@ -22,17 +23,14 @@ spec:
       labels:
         {{- include "capsule-proxy.selectorLabels" . | nindent 8 }}
     spec:
-      {{- if .Values.hostNetwork }}
```

Comment:
The removal of the `hostNetwork` is letting fail the e2e case, it should be put back.

Reply:
@prometherion run e2e test as `DaemonSet` with `hostNetwork=true` now succeeds.